testsrc/test262.properties

```diff
@@ -340,15 +340,9 @@ built-ins/Date
 
 # this is really; slow disable for now
 # built-ins/decodeURI
-    ! name.js
-    ! S15.1.3.1_A2.4_T1.js
-    ! S15.1.3.1_A5.2.js
```

Comment:
the idea of leaving the excludes in was to be able to re enable this without breaking, i suggest to leave it as is was before